tests/src/test/scala/akka/kafka/scaladsl/TransactionsSpec.scala

```diff
@@ -282,8 +282,8 @@ class TransactionsSpec extends SpecBase(kafkaPort = KafkaPorts.TransactionsSpec)
         control
       }
 
-      val controls: Seq[Control] = (0 until elements / batchSize).map {
-        x => {
+      val controls: Seq[Control] = (0 until elements / batchSize).map { x =>
```

Comment:
Intellij just hates formatting of this piece of code.
Every time I just open Intellij, it automatically removes newline...